test/Renci.SshNet.Tests/Classes/NetConfClientTest.cs

```diff
@@ -105,7 +107,9 @@ public void OperationTimeout_GreaterThanLowerLimit()
             {
                 Assert.IsNull(ex.InnerException);
                 ArgumentExceptionAssert.MessageEquals("The timeout must represent a value between -1 and Int32.MaxValue, inclusive.", ex);
+#if NETCOREAPP3_0_OR_GREATER
                 Assert.AreEqual("value", ex.ParamName);
+#endif
```

Comment:
On lower targets, `ex.ParamName` is going to be `""`. It would probably be better to specify `nameof` at the callsite and remove the `[CallerArgumentExpression]` attribute?